### PR TITLE
Update vehicles.lua

### PR DESCRIPTION
**Pull request by r0adra93**
fixex broken hashes and added comments to help people with the vehciles.lua

## Description

I added comments to help new people with vehicles.lua hopefully will help with the support questions.
I found some typos and fixed those.

## Checklist

- [x] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

**Review by KamuiKody — COMMENTED** [review]
i have tested these changes some of these vehicles i didnt even know existed! works good good work roadrage!

**Comment by r0adra93** [comment]
> i have tested these changes some of these vehicles i didnt even know existed! works good good work roadrage!

Great thanks! Care to approve the review?

**Review by KamuiKody — APPROVED** [review]

File: shared/vehicles.lua [hunk 1]
```diff
@@ -3,15 +3,15 @@ QBShared.VehicleHashes = {}
...
-        ['shop'] = {'pdm'},
+    ['asbo'] = { -- This is to match the spawn code of the vehicle
+        ['name'] = 'Asbo', -- this is a custom name of the car
+        ['brand'] = 'Maxwell', -- This is the vehicle's brand (ford, chevy, bmw, mercedes, etc)
+        ['model'] = 'asbo', -- this must match the spawn code of the vehicle
+        ['price'] = 4000, -- sales price of vehicle
+        ['category'] = 'compacts', -- this must match vehicleclass in vehicles.meta for the vehicle and must match with https://docs.fivem.net/natives/?_0x29439776AAA00A62
+        ['categoryLabel'] = 'Compacts', -- Customize for your VehicleShop headings (can even include spaces)
+        ['hash'] = `asbo`, -- must contain back ticks instead of single quotes and must match the spawn code of the vehicle
+        ['shop'] = {'pdm'}, -- can be a single shop or multiple shops. For multiple shops for example {'shopname1','shopname2','shopname3'}
```
Comment (Z3rio) [inline shared/vehicles.lua:14]:
```suggestion
    ['asbo'] = { -- This has to match the spawn code of the vehicle
        ['name'] = 'Asbo', -- This is the display name of the vehicle
        ['brand'] = 'Maxwell', -- This is the vehicle's brand (ford, chevrolet, bmw, mercedes, etc)
        ['model'] = 'asbo', -- This must match the spawn code of the vehicle
        ['price'] = 4000, -- The price that the vehicle sells for
        ['category'] = 'compacts', -- This must match "VehicleClass" in vehicles.meta for the vehicle and must also exist in https://docs.fivem.net/natives/?_0x29439776AAA00A62
        ['categoryLabel'] = 'Compacts', -- Customize for your Vehicle Shop headings (can even include spaces)
        ['hash'] = `asbo`, -- The hash key of the vehicle. Either backticks or GetHashKey can be used for generating this.
        ['shop'] = {'pdm'}, -- Can be a single shop or multiple shops. For multiple shops for example {'shopname1','shopname2','shopname3'}
```

Looks great, although I think these explenations would fit better! :)

File: shared/vehicles.lua [hunk 2]
```diff
@@ -5339,7 +5339,7 @@ QBShared.Vehicles = {
         ['shop'] = 'truck',
     },
     ['mixer'] = {
-        ['name'] = 'Cement Mixer',
+        ['name'] = 'HVY Cement Mixer',
```
Comment (Z3rio) [inline shared/vehicles.lua:5342]:
```suggestion
        ['name'] = 'Cement Mixer',
```

The brand is not needed in the actual label of the vehicle.

File: shared/vehicles.lua [hunk 3]
```diff
@@ -5349,7 +5349,7 @@ QBShared.Vehicles = {
         ['shop'] = 'truck',
     },
     ['mixer2'] = {
-        ['name'] = 'Mixer2',
+        ['name'] = 'HVY Cement Mixer 2',
```
Comment (Z3rio) [inline shared/vehicles.lua:5352]:
```suggestion
        ['name'] = 'Cement Mixer 2',
```
The brand is not needed in the actual label of the vehicle.

**Review by Z3rio — CHANGES_REQUESTED** [review]
Looks good, although I think these changes would make it even better! :)

File: shared/vehicles.lua (hunk 1 above)
Comment (r0adra93) [reply shared/vehicles.lua:14]: This part:
The hash key of the vehicle. Either backticks or GetHashKey can be used for generating this.
the people don't knwo what the hell GetHashKey is they aren't programmers they just want to get a server going.
Make it simple
This has to match the spawn code of the vehicle and must be surrounded by backticks. Example of a Backtick: `

**Comment by r0adra93** [comment]
Merged into https://github.com/qbcore-framework/qb-core/pull/883

**Pull request closed without merging**